### PR TITLE
Grouped Trace tabs

### PR DESCRIPTION
**Pull request by nicolaskruchten**
This PR replaces #160. Multi-value UX similar to Axes panel yet to come, in another PR :)

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
Oh I should mention that trace type names in a group context are not localized (or even properly capitalized!) yet but that will come once #311 is merged and I'll be able to leverage those localized constants from somewhere.

**Comment by nicolaskruchten** [comment]
I've added an item to #290 to capture this.

File: src/components/containers/Panel.js [hunk 1]
```diff
@@ -106,7 +108,7 @@ class Panel extends Component {
     );
 
     return (
-      <div className={bem('panel')}>
+      <div className={`panel${tabs ? ' panel--with-tabs' : ''}`}>
```
Comment (VeraZab) [inline src/components/containers/Panel.js]: yeah, I think we're not really using that bem stuff, we should probably remove it at some point..

Comment (nicolaskruchten) [reply src/components/containers/Panel.js]: This was added by @aulneau so we should get his input... we should either do one big pass where we remove it or add it everywhere in a single PR but agreed that we should standardize!

Comment (VeraZab) [reply src/components/containers/Panel.js]: no : ) this was added before aulneau, we've just all been adding to it, but yeah @aulneau what do you think about that bem utility, useful?

**Comment by VeraZab** [comment]
This is clean! I'm just going to check out this branch and test it out, but 💃

File: src/components/containers/Panel.js (hunk 1 above)
Comment (nicolaskruchten) [reply src/components/containers/Panel.js]: Oh, did you mean the `bem()` function or the actual BEM pattern?

And yes, the particular edit you're commenting on came from Thomas ;)

Comment (nicolaskruchten) [reply src/components/containers/Panel.js]: actually come to think of it I was going to refactor it a bit, good that you chose this one to comment on!

Comment (nicolaskruchten) [reply src/components/containers/Panel.js]: in the same family of functions as things like `bem()` I like https://github.com/JedWatson/classnames FWIW...

Comment (VeraZab) [reply src/components/containers/Panel.js]: I meant the `bem` function, which was introduced before @aulneau started contributing to this project : )

**Pull request merged**